corehq/apps/export/views.py

```diff
@@ -1673,6 +1675,16 @@ def _check_deid_permissions(self, export_instances):
                         "De-Identified export.")
                     )
 
+    def _check_export_size(self, export_instances, filters):
+        count = 0
+        for instance in export_instances:
+            count += get_export_size(instance, filters)
+        if count > MAX_EXPORTABLE_ROWS:
```

Comment:
Yeah. Could also change the error message if we want. Figured it would be good to give the user an idea of how many records they need to remove via filtering